### DISCUSSION TITLE
How can I add hourly to activities?

### DISCUSSION THREAD

**Discussion by gerroon** [kimai/kimai#5062] [Questions & Answers]
Hi

When I create an activity for a project only monthly is listed as budget type. Is there a way to add the hourly as budget type at all? thnx

**Comment by kevinpapst** [comment]
I don't know what "hourly as budget type" should be.

**Reply by gerroon** [reply]
Hi, please see the image. As you see only monthly is listed there
![image](https://github.com/user-attachments/assets/ec8bd649-158e-47de-86de-de265d7fb4ef)
